docs/team/leeenen.md

```diff
@@ -0,0 +1,51 @@
+---
+
+### Project: TAlent Assistant™
+
+TAlent Assistant™ is a desktop lightweight and centralized management system catered to professors for managing the
+process of hiring undergraduate/graduate Teaching Assistants (TA). They will be able to access the TAs’ application 
+data easily and review qualifications or availability for scheduling interviews. It is optimized for use via a Command 
+Line Interface (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, this 
+application will be able to help you manage all things under the hood of the TA initiative faster than traditional 
+GUI applications.
+
+Given below are my contributions to the project.
+
+* **New Feature**: Added the ability to undo/redo previous commands.
+    * What it does: allows the user to undo all previous commands one at a time. Preceding undo commands can be reversed by using the redo command.
+    * Justification: This feature improves the product significantly because a user can make mistakes in commands and the app should provide a convenient way to rectify them.
+    * Highlights: This enhancement affects existing commands and commands to be added in future. It required an in-depth analysis of design alternatives. The implementation too was challenging as it required changes to existing commands.
+    * Credits: *{mention here if you reused any code/ideas from elsewhere or if a third-party library is heavily used in the feature so that a reader can make a more accurate judgement of how much effort went into the feature}*
+
+* **New Feature**: Added a history command that allows the user to navigate to previous commands using up/down keys.
+
+* **Code contributed**: [RepoSense link]()
+
+* **Project management**:
+    * Managed releases `v1.3` - `v1.5rc` (3 releases) on GitHub
+
+* **Enhancements to existing features**:
+    * Updated the GUI color scheme (Pull requests [\#33](), [\#34]())
+    * Wrote additional tests for existing features to increase coverage from 88% to 92% (Pull requests [\#36](), [\#38]())
+
+* **Documentation**:
+    * User Guide:
+        * Added documentation for the features `delete` and `find` [\#72]()
+        * Did cosmetic tweaks to existing documentation of features `clear`, `exit`: [\#74]()
+    * Developer Guide:
+        * Added implementation details of the `delete` feature.
+
+* **Community**:
+    * PRs reviewed (with non-trivial review comments): [\#12](), [\#32](), [\#19](), [\#42]()
+    * Contributed to forum discussions (examples: [1](), [2](), [3](), [4]())
+    * Reported bugs and suggestions for other teams in the class (examples: [1](), [2](), [3]())
+    * Some parts of the history feature I added was adopted by several other class mates ([1](), [2]())
+
+* **Tools**:
+    * Integrated a third party library (Natty) to the project ([\#42]())
+    * Integrated a new Github plugin (CircleCI) to the team repo
+
+* _{you can add/remove categories in the list above}_
```

Comment:
EE, I think can clean up but leave the headers there. IMO!

Reply:
I agree, will make the necessary changes. 👍